spki/src/spki.rs

```diff
@@ -57,3 +68,24 @@ impl<'a> Message<'a> for SubjectPublicKeyInfo<'a> {
         f(&[&self.algorithm, &BitString::new(self.subject_public_key)?])
     }
 }
+
+#[cfg(feature = "fingerprint")]
+#[cfg_attr(docsrs, doc(cfg(feature = "fingerprint")))]
+impl<'a> SubjectPublicKeyInfo<'a> {
+    const BUFSIZE: usize = 4096;
+
+    #[cfg(all(feature = "fingerprint", feature = "alloc"))]
+    #[cfg_attr(docsrs, doc(cfg(all(feature = "fingerprint", feature = "alloc"))))]
```

Comment:
IIRC `doc(cfg(...))` doesn't like this sort of nesting and you need to include two directives, but I'd have to double check how it renders locally. You can do that with something like:

```
$ cargo +nightly rustdoc --all-features --open -- --cfg docsrs
```

Reply:
I did check that before committing, it renders ok. If two directives is the preferred style I'll happily change

Reply:
Oh hey, it does work! Great. Never mind then.

<img width="741" alt="Screen Shot 2021-09-16 at 5 03 12 PM" src="https://user-images.githubusercontent.com/797/133696410-9b5ae979-c7d6-4f65-9fc5-7d996b99bccb.png">